Entities/Common/GUI/WheelMenuCommon.as

```diff
@@ -11,7 +11,7 @@ namespace WheelMenu
 	const SColor pane_title_color(0xFFCCCCCC);
 	const SColor pane_text_color(0xFFFFFFFF);
 
-	const float item_distance = 0.4f;
+	const float item_distance = 0.25f;
```

Comment:
Changing this constant effects all while menus in the game. Instead make the WheelMenu base class have an item_distance member that defaults to this constant (0.4) and override it in the PickupWheelMenu

Reply:
Woops. Fixed. Might be interesting to have as option in settings. 
Not sure how to do that though.

Reply:
You can create custom menus for options (see emote binding menu) and you can save/load preferences to cfg files. Though for this right now I wouldn't recommend it.

Probably better solved with proper UI scaling.